packages/client/internals/Controls.vue

```diff
@@ -3,20 +3,22 @@ import { shallowRef } from 'vue'
 import { showInfoDialog, showRecordingDialog } from '../state'
 import { configs } from '../env'
 import { useNav } from '../composables/useNav'
+import { useFeatures } from '../composables/useFeatures'
 import QuickOverview from './QuickOverview.vue'
 import InfoDialog from './InfoDialog.vue'
 import Goto from './Goto.vue'
 import ContextMenu from './ContextMenu.vue'
 
 const { isEmbedded } = useNav()
-const drawingEnabled = __SLIDEV_FEATURE_DRAWINGS__ && !configs.drawings.presenterOnly && !isEmbedded.value
+const features = useFeatures()
+const drawingEnabled = features.drawings && !configs.drawings.presenterOnly && !isEmbedded.value
 const DrawingControls = shallowRef<any>()
 if (drawingEnabled)
   import('../internals/DrawingControls.vue').then(v => DrawingControls.value = v.default)
 
 const WebCamera = shallowRef<any>()
 const RecordingDialog = shallowRef<any>()
-if (__SLIDEV_FEATURE_RECORD__) {
+if (features.record) {
```

Comment:
The reason we use the flag and static replacement is for tree-shaking. When the feature is disabled, the corresponding module should not be shipped to user's client bundle at all because Rollup will remove `if (false) {}`

Is the new approach still behaviours like that?